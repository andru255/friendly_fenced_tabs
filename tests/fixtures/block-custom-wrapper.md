```bash friendly_attrs={'class':'my-custom-classname', 'data-foo'='bar'}
echo "An example with title"
```