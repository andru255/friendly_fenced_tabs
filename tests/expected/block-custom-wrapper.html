<div class="my-custom-classname"
     data-foo="bar">
    my example of block
</div>
